protocols/tests/test_migration/test_migration_reports_3_0_0_to_reports_4_0_0.py

```diff
@@ -5,22 +5,21 @@
 from protocols.reports_3_0_0 import ReportedSomaticVariants as ReportedSomaticVariants_old
 from protocols.reports_4_0_0 import ReportEventCancer as ReportEventCancer_new
 from protocols.reports_4_0_0 import ReportedSomaticVariants as ReportedSomaticVariants_new
-from protocols.util import get_valid_reported_somatic_variant_3_0_0
-from protocols.util import get_valid_reported_somatic_variant_4_0_0
-from protocols.util.generate_mock_objects import get_valid_report_event_cancer_3_0_0
+from protocols.util.dependency_manager import VERSION_300, VERSION_400
```

Comment:
Can we please change this line to:
```
from protocols.util.dependency_manager import VERSION_400
from protocols.util.dependency_manager import VERSION_300
```
or 
```
from protocols.util import dependency_manager
```
and refer to `dependency_manager.VERSION_400` and `dependency_manager.VERSION_300`